index.js

```diff
@@ -1,13 +1,22 @@
-const { parse, valid } = require('./lib/input.js');
-const { execute } = require('./lib/note.js');
+const Input = require('./Input.js');
+const Notes = require('./Notes.js');
+
+const input = new Input(process.argv);
+if(input.valid()) Notes.execute(input);
+else console.log('invalid command');
+
+
 
 // use parse to create an action from process.argv
```

Comment:
delete this stuff. you have a branch so you can always see the old way

Reply:
Thanks, I'll do that moving forward!